packages/components/data-table/src/cell.styles.js

```diff
@@ -172,11 +173,18 @@ const SortableHeaderInner = styled.button`
   ${getSortableHeaderStyles}
 `;
 
+const CellCollapseButtonWrapper = styled.div`
```

Comment:
I think we should try to make this control an accessible `button` element, and also with a hover/focus effect (like our [SecondaryIconButton](https://uikit.commercetools.com/?path=/story/components-buttons--secondaryiconbutton)), so that the user has a better perception of whether he's clicking the row or the little button, and avoiding accidental clicks on one or the other.